### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Changes:
- Add blog link
- Beautification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 
-## Contribute 
-- Feel free to open an issue or recommend any changes which you want to see in the website. 
+## Contribute to the Website 
+- Feel free to open an issue or recommend any changes which you want to see on the website. 
 - Adhere to the guidelines mentioned below if you want to contribute. 
-- To Contribute, check out official [wiki](https://github.com/asetalias/asetalias.github.io/wiki)
+- Before contributing, be sure to check out the official [wiki](https://github.com/asetalias/asetalias.github.io/wiki)
+
+## Contirbute to ALiAS Writes's blog!
```
Comment (Sc0rpi0n101) [inline README.md]:
```suggestion
## Contribute to our blog: ALiAS Writes!
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+
+ALiAS has its own blog now, [ALiAS Writes](https://asetalias.in/blog). If you like your contributions to be more content, than technical then we we would love to see your blogs be published on ALiAS Writes. [Read more about it here on How to Contibute](https://github.com/asetalias/blog/blob/master/README.md).
```
Comment (Sc0rpi0n101) [inline README.md]:
```suggestion
ALiAS has its own blog now, [ALiAS Writes](https://asetalias.in/blog). If you like your contributions to be more content-centric than technical, we would love to see your blogs on ALiAS Writes. Read more about How to Contribute [here](https://github.com/asetalias/blog/blob/master/README.md). 
```

Comment (Sc0rpi0n101) [reply README.md]: I feel something like content-centric, content-oriented or content-based is better than just content

File: README.md [hunk 3]
```diff
@@ -16,20 +30,11 @@
...
-- @bhanuvrat
-- @ATB00ker
-- @vipulgupta2048
-- @omi10859
-- @Manas-Kashyap
-- @JoyBhalla
-- @akshat0047
-- @vips28
-- @utkarsh2102
+Thanks to all the generous contributions both offline and online by our members, is behind the magic of everything that ALiAS does.
```
Comment (Sc0rpi0n101) [inline README.md]: I feel like this one doesn't flow well
```suggestion
A big thanks to all the generous contributions both offline and online by our members, who are behind the magic of everything that ALiAS does.
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,26 @@
 ## ALiAS: Amity Linux Assistance Sapience
-- The following repo contains the source code for the official website of Amity Linux Assistance Sapience. 
+
+ [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![GitHub license](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE)
```
Comment (Sc0rpi0n101) [inline README.md]: Why 'GitHub' License?
```suggestion
 [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![License](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 
```

File: README.md [hunk 5]
```diff
@@ -1,12 +1,26 @@
...
+![GitHub issues](https://img.shields.io/github/issues-raw/asetalias/asetalias.github.io.svg) ![GitHub repo size](https://img.shields.io/github/repo-size/asetalias/asetalias.github.io.svg)
+![Twitter Follow](https://img.shields.io/twitter/follow/asetalias.svg?style=social)
+
+
+- The following repository contains the source code for the official website of Amity Linux Assistance Sapience. The largest open-source community of Amity University in Noida and now also in Lucknow. 
 - The website is hosted on gh-pages itself. 
-- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML Files.
+- This Website uses Staticjinja to build top-level HTML files. Follow Instructions to modify the HTML files.
+
+<img src="https://user-images.githubusercontent.com/22801822/67155846-3d12c500-f334-11e9-9125-dbf32ea573a9.png" height=480>
```
Comment (Sc0rpi0n101) [inline README.md]: I can see the exit full-screen prompt.

File: README.md (hunk 1 above)
Comment (vipulgupta2048) [reply README.md]: Good catch!

File: README.md (hunk 4 above)
Comment (vipulgupta2048) [reply README.md]: Copied it from the blog README, didn't give it a second look.

File: README.md (hunk 5 above)
Comment (vipulgupta2048) [reply README.md]: Oh, that's embarrassing. Let me add a new one.

**Comment by vipulgupta2048** [comment]
Houston: We are ready for launch :rocket:

**Pull request merged**